explorer/client/src/reducers/config.ts

```diff
@@ -0,0 +1,21 @@
+export interface IState {
+  etherscanHost?: string
+}
+
+export type Action =
+  | { type: 'UPSERT_JOB_RUN'; data: any }
+  | { type: '@@redux/INIT' }
+  | { type: '@@INIT' }
+
+const initialState = { etherscanHost: undefined }
+
+export default (state: IState = initialState, action: Action) => {
+  switch (action.type) {
+    case 'UPSERT_JOB_RUN':
+      return Object.assign({}, state, {
+        etherscanHost: action.data.meta.jobRun.meta.etherscanHost
```

Comment:
Two things:
1. Reiterate the long combination of keys: ` action.data.meta.jobRun.meta.etherscanHost`
2. Is `UPSERT_JOB_RUN` really the only action type we need to cover. If we aren't mutating, but merely retrieving, don't we still need the etherscanHost? Or is that all covered in this one action `UPSERT_JOB_RUN`? Seems so when I peruse https://github.com/smartcontractkit/chainlink/blob/master/explorer/client/src/reducers/jobRuns.ts#L25

Reply:
1. `UPSERT_...` in this context refers to inserting or updating the job run in the redux store. After the store has the job run, it gets retrieved by a selector for display.
2. So far yes. If we wanted to show a link on the search results index for example, we would need to change the approach.